ansible/roles/artifacts/tasks/main.yaml

```
@@ -530,6 +530,45 @@
     mode: "644"
     checksum: sha256:215493daac110055990318b7916fbf1a7d6b08feea9c27cdd6403df31557d992
 
+# tensorrt_vad
+- name: Create tensorrt_vad directory inside {{ data_dir }}
+  ansible.builtin.file:
+    path: "{{ data_dir }}/vad"
+    mode: "755"
+    state: directory
+
+- name: Download tensorrt_vad/vad-carla-tiny_backbone.onnx
+  become: true
+  ansible.builtin.get_url:
+    url: https://awf.ml.dev.web.auto/planning/models/tensorrt_vad/carla_tiny/v0.1/vad-carla-tiny_backbone.onnx
+    dest: "{{ data_dir }}/vad/vad-carla-tiny_backbone.onnx"
```

```suggestion
    dest: "{{ data_dir }}/vad/v1.0/vad-carla-tiny_backbone.onnx"
```
If you plan to update versions in the future, you need to include version numbers in the directory name; otherwise, Ansible won’t be able to perform the update.

---

@yukkysaito 
Thank you for the comments. 
Already fixed in  https://github.com/autowarefoundation/autoware/pull/6557/commits/459d5319ddd99579001457dea2eda4b6d211efbc and https://github.com/autowarefoundation/autoware_universe/pull/11578/commits/a4b9f61a75e088d98c6fe1ea92af759b540c35a7